### ISSUE TITLE
Futute @nuxtjs/google-fonts

### ISSUE THREAD

**Issue by ricardogobbosouza** [nuxt/fonts#57]
Hi @danielroe 

Should we deprecate the `@nuxtjs/google-fonts` module in favor of this module?
It is a widely used module, do I continue to support it?
Should I add a note indicating the use of `nuxt/fonts` ?

**Comment by danielroe** [comment]
You've done an amazing job with `@nuxtjs/google-fonts` and I very much hope you'll be able to help maintain `@nuxt/fonts` as well, to the extent you have time.

Having said that, I would suggest:
* add a note pointing to `@nuxt/fonts` with some migration steps so we can get feedback and improve `@nuxt/fonts`
* adding issues so we can track missing features or where `@nuxt/fonts` needs to improve to be on par with `@nuxtjs/google-fonts`
* continuing to support `@nuxtjs/google-fonts` until we both feel that `@nuxt/fonts` is mature enough to be a straightforward and bug-free migration (maybe we could indicate this with a v1 release?)

I would very much welcome any thoughts and feedback from you on this. What do you think?

**Comment by ricardogobbosouza** [comment]
I agree, once on v1 we should create a migration section for nuxt/fonts

**Comment by danielhill1982** [comment]
I was forcing an issue with nuxt google-font module.

I just use this now, too.
It is a very nice module to quickly add webfonts, like google fonts!

Thanks.



[github.com/danielhill1982](https://github.com/danielhill1982)
